arangod/IResearch/IResearchLink.cpp

```diff
@@ -1489,7 +1491,8 @@ arangodb::Result IResearchLink::properties( // get link properties
     arangodb::velocypack::Builder& builder, // output buffer
     bool forPersistence // properties for persistance
 ) const {
-  if (!builder.isOpenObject() || !_meta.json(builder, forPersistence)) {
+  if (!builder.isOpenObject() // not an open object
```

Comment:
perhaps this should be an assertion?

Reply:
in other places we always provide a Result() for such cases